backend/sites/src/app/services/folio/folioContents.service.ts

```diff
@@ -0,0 +1,108 @@
+import { Injectable } from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+import { FolioContents } from '../../entities/folioContents.entity';
+import { Repository } from 'typeorm';
+import { FolioContentDTO } from '../../dto/folioContent';
+import { plainToInstance } from 'class-transformer';
+
+@Injectable()
+export class FolioContentsService {
+  constructor(
+    @InjectRepository(FolioContents)
+    private folioContentRepository: Repository<FolioContents>,
+  ) {}
+
+
+  async getSiteForFolio(folioId: string): Promise<FolioContents[]> {
+    try {
+      const sitesForFolioArr = await this.folioContentRepository.find({  
+        relations: ['site','folio'],
+        where: { folioId },
+      });
+      return sitesForFolioArr;
+    } catch (error) {
+      throw error;
```

Comment:
can we have consistency in all service layer error handling ?

Reply:
done